README.md

```diff
@@ -32,6 +32,14 @@ Altenatively one can use the embedding API, see [src/mount.coffee](./src/mount.c
     rt.start (err, rt) ->
       # started
 
+With NoFlo 0.7.0, component discovery and caching 
```

Comment:
It should mostly just refer to documentation in grunt-noflo-browser, but this does not exist, https://github.com/noflo/grunt-noflo-browser/issues/8